src/main/java/calculator/validator/ConsecutiveLetterValidator.java

```diff
@@ -0,0 +1,31 @@
+package calculator.validator;
+
+public class ConsecutiveLetterValidator {
+
+
+    public ConsecutiveLetterValidator(String input) {
+
+        validateConsecutiveLetters(input);
+    }
+
+    private void validateConsecutiveLetters(String input) {
+        if (input.charAt(0) != '/' && checkConsecutiveLetter(input)) {
+            throw new IllegalArgumentException("같은 문자가 두 번 연속 입력되었습니다.");
+        }
+        if (input.charAt(0) == '/' && checkConsecutiveLetter(input.substring(5))) {
+            throw new IllegalArgumentException("같은 문자가 두 번 연속 입력되었습니다.");
+        }
```

Comment:
커스텀 구분자가 정의된 경우와 정의되지 않은 경우를 구분한 것 같은데, 조금 가시성이 떨어지는 것 같아요 🥹
이렇게 상수화해서 묶는 방향은 어떻게 생각하시는지 궁금합니다!
```suggestion
        if (input.charAt(0) == '/' && checkConsecutiveLetter(input.substring(NUMBERS_START_INDEX))) {
            throw new IllegalArgumentException("같은 문자가 두 번 연속 입력되었습니다.");
        }
```

Reply:
상수를 잘 활용한 좋은 예시 같아요.